cibuildwheel/linux.py

```diff
@@ -180,7 +180,7 @@ def build_in_container(
         log.step("Running before_all...")
 
         env = container.get_environment()
-        env["PATH"] = f'/opt/python/cp38-cp38/bin:{env["PATH"]}'
+        env["PATH"] = f'/opt/python/cp39-cp39/bin:{env["PATH"]}'
```

Comment:
This is strictly connected to this Pr or only change required by manylinux?

Reply:
It should probably be a separate PR. I will do this.